documentation/bolt_examples.md

```diff
@@ -123,3 +123,215 @@ To convert the `restart_service.ps1` script to a task, giving you the ability to
     ![](bolt_PS_3.png)
 
     **Note:** `--nodes windows` refers to the name of the group of target nodes that you specified in your inventory file. For more information, see [Specify target nodes](bolt_options.md#).
+
+
+## Deploy a package with Bolt and Chocolatey
+
+You can use Bolt with Chocolatey to deploy a package on a Windows node by writing a Puppet plan that installs Chocolatey and uses the Chocolatey package provider to install a package. This is all done using content from the Puppet Forge.
+
+**Before you begin** 
+
+- Install Bolt and Puppet Development Kit (PDK).
+- Ensure you have Powershell and Windows Remote Management (WinRM) access.
+
+In this example, you:
+
+- Build a project-specific configuration using a Bolt project directory and PDK.
+- Download module content from the Puppet Forge.
+- Write a Bolt plan to apply Puppet code and orchestrate the deployment of a package resource using the Chocolatey provider.
+
+### 1. Build a project-specific configuration using a Bolt project directory because Bolt runs in the context of a project directory. This directory contains all of the configuration, code, and data loaded by Bolt.
+
+1. Create a directory called `puppet_choco_tap` and add a bolt.yaml file. Whenever you run Bolt inside this project directory, it will load the code and configuration.
```

Comment:
```suggestion
1. Create a directory called `puppet_choco_tap` and add a `bolt.yaml` file. Whenever you run Bolt inside this project directory, it will load the code and configuration.
```

Reply:
It'd be nice if, below this line, there were the specific commands to run to create the directory.

```
mkdir puppet_choco_tap
New-Item -Type File -Path .\puppet_choco_tap\bolt.yaml
```